synapse/cores/common.py

```diff
@@ -377,7 +377,7 @@ def _onSetCaching(self, valu):
 
     def _reqSpliceInfo(self, act, info, prop):
         valu = info.get(prop)
-        if prop == None:
+        if valu == None:
```

Comment:
I made this same argument when I first started working on synapse, but @invisig0th had some prior experience with some sort of python bug where the None reference wasn't always the same?